packages/teams-js/src/public/interfaces.ts

```diff
@@ -1262,3 +1262,41 @@ export interface ClipboardParams {
   /** Blob content in Base64 string format */
   content: string;
 }
+
+/**
+ * @hidden
+ * List of enter focus action items
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export enum EnterFocusActionItem {
+  /**
+   * Determines the direction to focus in app.
+   */
+  PreviousLandmark,
+  NextLandmark,
+  /**
+   * Determines if the focus should go to the particular content of the app.
+   */
+  Read,
+  Compose,
+}
+
+/**
+ * Return focus action items
+ */
+export enum ReturnFocusActionItem {
+  /**
+   * Determines the direction to focus in host for previous landmark.
+   */
+  PreviousLandmark,
```

Comment:
maybe Landmark is implicit and so can the enum value be Previous, Next, ... ?

Reply:
I would like to keep it as PreviousLandmark and NextLandmark because in future there might be additional values could be populated here which aren't landmark